wallet/wallet.go

```diff
@@ -2524,6 +2524,57 @@ func (w *Wallet) GetTicketsPrecise(f func([]*TicketSummary, *wire.BlockHeader) (
 	return nil
 }
 
+// GetTicketInfo returns the ticket summary and the corresponding block header
+// for the provided ticket. The ticket summary comprises of the transaction
+// summmary for the ticket and the spender (if already spent) and the ticket's
+// current status.
+func (w *Wallet) GetTicketInfo(chainClient *dcrrpcclient.Client, hash *chainhash.Hash) (*TicketSummary, *wire.BlockHeader, error) {
+	const op errors.Op = "wallet.GetTicketInfo"
+
+	ticketSummary := new(TicketSummary)
+	blockHeader := new(wire.BlockHeader)
+
+	err := walletdb.View(w.db, func(dbtx walletdb.ReadTx) error {
+		txmgrNs := dbtx.ReadBucket(wtxmgrNamespaceKey)
+
+		// Fetch the transaction details for the provided hash.
+		txDetail, err := w.TxStore.TxDetails(txmgrNs, hash)
+		if err != nil {
+			return err
+		}
+
+		// Assert the transaction type is that of a ticket.
```

Comment:
Personally, I feel this type of comment is unnecessary given that it's only describing _what_ the code below is doing vs _why_ and the relevant code is pretty obvious given how it's written and the underlying semantics.

You don't need to change this on my account, though! Just wanted to add my perspective...